### PR TITLE
Sponsors

### PR DESCRIPTION
**Pull request by pedromsfernandes**
# Description

Added sponsors from this year, and removed images from last year. Also, after checking the mock-ups for the home page, and remembering feedback from last year (companies weren't happy that their logo wasn't being shown at all times), I decided to remove the carousel from the sponsors section. I'd like to get your feedback on this.

# Tasks

- [x] Remove sponsor images from last year
- [x] Add sponsors available so far
- [x] Remove carousel from sponsors section
- [x] Add BPI (logo is in .ai)
- [ ] Add Jumia and Deloitte (logos not available)

### CODE REVIEW

**Comment by MPDR200011** [comment]
There is a mistake with the gold sponsor, it is BPI not Accenture.

Also, I find it hard to differentiate between the different tiers of sponsors as they are all inside the same white background.

**Comment by pedromsfernandes** [comment]
> There is a mistake with the gold sponsor, it is BPI not Accenture.

According to the drive Accenture and BPI are both gold sponsors.

> Also, I find it hard to differentiate between the different tiers of sponsors as they are all inside the same white background.

The mock-ups had the tiers differentiated by size and space between tiers. We can bring back the tier title we had last year, but I'd leave that decision to @SantosAntero.

File: src/data/events/visits/pixelmatters.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (SocksDevil) [inline src/data/events/visits/pixelmatters.md]: Should these visits markdowns still be here?

**Review by SocksDevil — CHANGES_REQUESTED** [review]

File: src/data/events/visits/pixelmatters.md (hunk 1 above)
Comment (pedromsfernandes) [reply src/data/events/visits/pixelmatters.md]: They are deleted in the program PR

**Review by SocksDevil — DISMISSED** [review]
:ship:

**Comment by MPDR200011** [comment]
> According to the drive Accenture and BPI are both gold sponsors.

Oh ok, I was concerned since Accenture still wasn't announced in social media, is it expected to be announced before the website launches?

**Comment by SocksDevil** [comment]
Even if it isn't, there isn't any problem with deploying it as it is, it's
no secret :smiley:

**Review by MPDR200011 — DISMISSED** [review]

**Review by SocksDevil — DISMISSED** [review]

**Review by MPDR200011 — DISMISSED** [review]

**Review by SocksDevil — APPROVED** [review]
:rocket:

**Review by MPDR200011 — APPROVED** [review]

**Pull request merged**